shared/lib_battery_dispatch.h

```diff
@@ -48,7 +45,7 @@ class dispatch_t
 public:
 
 	enum FOM_MODES { FOM_LOOK_AHEAD, FOM_LOOK_BEHIND, FOM_FORECAST, FOM_CUSTOM_DISPATCH, FOM_MANUAL, FOM_RESILIENCE };
-	enum BTM_MODES { LOOK_AHEAD, LOOK_BEHIND, MAINTAIN_TARGET, CUSTOM_DISPATCH, MANUAL, RESILIENCE };
+	enum BTM_MODES { LOOK_AHEAD, LOOK_BEHIND, MAINTAIN_TARGET, CUSTOM_DISPATCH, MANUAL, FORECAST, RESILIENCE };
```

Comment:
Is RESILIENCE dependent on the enum order anywhere? Typically, we add new enums to the end to avoid conflict...

Reply:
I just did a search for _mode == [number] and didn't see anything. The only place where these are referenced by number is with the radio button in the SAM GUI, where resilience isn't an option. I placed FORECAST where I did so that the radio button went 0-5, instead of 0-4,6.

Reply:
👍 They'll be removed in the next push.